### PR TITLE
Update documentation for Caddy reverse proxy with subfolder

### PR DESCRIPTION
**Pull request by soniyadotp**
Closes_ #6208

Pull request checklist:

- [ ] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [x] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

**Comment by Frenzie** [comment]
I think you meant to put that in `en` in addition to `fr`?

**Comment by soniyadotp** [comment]
So, should I add this in en folder as well?

**Comment by Frenzie** [comment]
Yes please.

**Comment by soniyadotp** [comment]
I have added in en folder as well. Thanks :)

**Review by Frenzie — APPROVED** [review]
I don't know Caddy, but looks sane enough to me.

**Comment by soniyadotp** [comment]
> I don't know Caddy, but looks sane enough to me.

The Markdown syntax has been revised to address the issues identified during the automated checks.

File: docs/en/developers/03_Backend/06_Reverse_Proxy_Setup.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            header_up X-Forwarded-Host {host}
+            header_up X-Forwarded-For {remote}
+            header_up X-Forwarded-Ssl {on}
+            header_up X-Forwarded-Prefix "/freshrss/"
+        }
+    }
+}
+```
+
+Replace `yourdomain.com` with your actual domain and `freshrss` with the subfolder name where FreshRSS is hosted.
```
Comment (Alkarex) [inline docs/en/developers/03_Backend/06_Reverse_Proxy_Setup.md]:
```suggestion
Replace `example.net` with your actual domain and `freshrss` with the subfolder name where FreshRSS is hosted.
```

Please use one of the domains reserved for examples / documentations, so we do not start linking to random existing sites: https://en.wikipedia.org/wiki/Example.com

**Comment by Alkarex** [comment]
We should add a link to those pages from one of the other documentation pages

**Comment by soniyadotp** [comment]
@Alkarex Could you please review or provide more detailed elaboration on what specific changes are needed in the current version?

**Comment by Alkarex** [comment]
https://freshrss.github.io/FreshRSS/en/admins/Caddy.html
linked from https://freshrss.github.io/FreshRSS/en/admins/01_Index.html

**Pull request merged**